src/components/Blog/BlogHeader.tsx

```diff
@@ -9,11 +9,11 @@ interface Props extends Pick<BlogTypeInfo, 'title' | 'belowTitle' | 'baseUrl'> {
 
 export const BlogHeader: FunctionComponent<Props> = ({ title, belowTitle, baseUrl }) => (
     <div>
-        <h1 className="display-4 font-weight-bold mt-3">
+        <h1 className="mt-3">
```

Comment:
Just flagging, do we want the header of the index this big? Seems like the blog may need a refresh anyway. This layout is the same on Podcast index too. The title seems extra large & doesn't line up with the cards anymore.
<img width="799" alt="image" src="https://user-images.githubusercontent.com/59381432/183736584-c758fc1f-2845-478d-9b03-db6aedaddfab.png">

Reply:
It's technically an h1 and that's what the DLS has defined them as, but you're right, it's quite big compared to what we have now. This would probably look much better with some vertical padding and a background, to look more like a proper hero section. I'm going to leave the sizing for now for our post-merge-audit, but I've fixed the alignment.